src/jit/compiler.cpp

```diff
@@ -1138,6 +1138,15 @@ var_types Compiler::getReturnTypeForStruct(CORINFO_CLASS_HANDLE clsHnd,
     {
         *wbReturnStruct = howToReturnStruct;
     }
+
+#ifdef _TARGET_ARM64_
```

Comment:
I think this should eventually be removed so normalization is enabled for all platforms, but I didn't want to have to debug any issues this would introduce.

Reply:
I believe the normalization is only required for `FEATURE_MULTIREG_RET`, and on x64/UX it isn't needed because it calls `eeGetSystemVAmd64PassStructInRegisterDescriptor()`. So I think this is only needed at most for ARM and ARM64, and it's not an issue for ARM at present because `FEATURE_SIMD` isn't supported yet. Would you mind adding a comment to that effect?

Reply:
@CarolEidt Added comment per your request